kubernetes/omsagent.yaml

```diff
@@ -765,7 +765,7 @@ spec:
         component: oms-agent-win
         tier: node-win
       annotations:
-        agentVersion: "1.10.0.1"
+        agentVersion: "azure-mdsd-1.14.2"
```

Comment:
we dont have AMA dependency yet for windows agent so i think this can be removed and added back when we get that integration. most likely version will be different for windows and linux.

Reply:
I am not sure what the impact would be if we removed this annotation key from the aks-rp chart. At the same time if it has the 1.10.0.1 version it would create confusion. So, I updated it to the new one. We can update this to the version based on what we use for windows later.